com.ticktick.TickTick.metainfo.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>com.ticktick.Ticktick</id>
+  <launchable type="desktop-id">com.ticktick.Ticktick.desktop</launchable>
+  <name>TickTick</name>
+  <project_license>LicenseRef-proprietary</project_license>
+  <developer_name>TickTick Team</developer_name>
+  <summary>TickTick: Todo list, checklist and task manager app</summary>
+  <description>
+    <p>Join millions of people to capture ideas, organize to-dos, and make the most of your life with TickTick.</p>
+    <p>Available for free, syncs across iOS, Android, Mac, Windows, Linux and Web.</p>
```

Comment:
This should include a disclaimer like https://flathub.org/apps/details/com.discordapp.Discord does.